system/Validation/FormatRules.php

```diff
@@ -333,6 +333,10 @@ public function valid_url_strict(?string $str = null, ?string $validSchemes = nu
      */
     public function valid_date(?string $str = null, ?string $format = null): bool
     {
+        if (($str === null)) {
```

Comment:
Why the need for additional parentheses?

Reply:
Oh, just a mistake.

Reply:
Removed.